PlayCover/Utils/PlayTools.swift

```diff
@@ -171,15 +171,7 @@ class PlayTools {
             let pluginUrl = exec.deletingLastPathComponent()
                 .appendingPathComponent("PlugIns")
 
-            let bundleTarget = pluginUrl
-                .appendingPathComponent("AKInterface")
-                .appendingPathExtension("bundle")
-
-            if FileManager.default.fileExists(atPath: bundleTarget.path) {
-                try FileManager.default.removeItem(at: bundleTarget)
-            }
-
-            if try FileManager.default.contentsOfDirectory(atPath: pluginUrl.esc).isEmpty {
+            if FileManager.default.fileExists(atPath: pluginUrl.path) {
```

Comment:
The PlugIn directory shouldn't be directly removed, since some apps use it.

Reply:
But then again, some apps don't have the directory, so it would need to be deleted if it was created by PlayCover.